docs/authoring.md

```diff
@@ -209,7 +223,10 @@ The `.tp-config.json` file contains a JSON object that may have the following in
   "generator": "<2.0.0",
   "filters": [
     "@asyncapi/generator-filters"
-  ]
+  ],
+  "hooks": {
+    "@asyncapi/generator-hooks": "hookFunctionName"
```

Comment:
Will it be better to provide upper in example section at least one hook with defined name?

Reply:
What do you mean? in example section we provide examples of hooks that you rather put locally in your template, writing own hooks library is like advanced usage in my opinion

this config example shows usage of our official hooks library, you do not have to add to config your local hooks

Reply:
Yes, but assume i'm newbie. I just read Hooks section, now read this section and see `specify exact name of the hook`. And now i ask, what exactly `hook name` is? This has not been mentioned before.
So, sure, you do not need to give names to local hooks, but example of hook with name will increase understanding of the concept, from my point of view.

Reply:
how about now? I added some more content https://github.com/asyncapi/generator/pull/334/commits/87a9dfd4f170a5ac8927c01f0e789f9fc13bdde9

Reply:
I still think that one example is better than long description 😄 , but nevertheless questions are described now in docs. So it's fine.